micromouse/Vector.cpp

```diff
@@ -10,7 +10,7 @@ namespace Micromouse
 
 	direction operator-(const direction &dir1, const direction &dir2)
 	{
-		return static_cast<direction>(((int)dir1 - (int)dir2) % 8);
+		return static_cast<direction>((64 + (int)dir1 - (int)dir2) % 8);
```

Comment:
@Pastyguy : "What the hell does it do?"


it was the bug that cost us 3 hours